camelAssemblies/build.gradle

```diff
@@ -162,3 +162,6 @@ tasks.register('publishAssemblies') {
     }
 }
 
```

Comment:
Allow a `gradle assemble importAssemblies publishAssemblies` command.  Takes a while for the fully bunch, so nice to fire & forget.